.lgtm.yml

```diff
@@ -0,0 +1,6 @@
+queries:
```

Comment:
Please remove this file, we don't anticipate using this service right now.

Reply:
No problem. I've removed this file.